marklogic-data-hub/src/main/java/com/marklogic/hub/DataHubUser.java

```diff
@@ -0,0 +1,114 @@
+package com.marklogic.hub;
```

Comment:
It looks like this file is only used by the unit tests. I'd rather it not be part of the publicly exposed Hub Jar since it's only purpose is to support a test case. Also, it might confuse Hub users into thinking it's a class they can/should use.